### DISCUSSION TITLE
Feature request: Condense repeat hosts in search results.

### DISCUSSION THREAD

**Discussion by BarkinMad** [Hunter-How/Support#11] [Q&A]
A lot of queries yield repeat results, taken at different times or with different domains making the search process tedious. 

Example:
    Query: as.org="Verizon"
    Results:
        10.10.10.10 | xxx.com @2024/01/02
        10.10.10.10 | yyy.com @2024/01/02
        10.10.10.10 | zzz.com @2024/01/02
        10.10.10.10 | xxx.com @ 2024/01/01
        
Modified results after this change:
    Query: as.org="Verizon"
    Results:
        10.10.10.10 | xxx.com @2024/01/02 (multiple results click for all)
        
Would help with filtering out honeypots that take up multiple pages of results. Please and thank you.

**Comment by HowHunterHow** [comment]
When we store data, we consider [1IP+1domain+1open port] as a service. 
In order to help many users understand how many different services are running on a single IP, the display method is adopted.
The honeypot issue can be frustrating to users like you. however, sometimes ppl are trying to find honeypot through our platform as well. 
Here is a little trick that may help with your problem. Try out our special query [portcount].

Example:
Query: as.org="Verizon"&&ip.port_count<"20"
Hopefully, this can eliminate those honeypot results that bother you. 

If you would like to search IP results without multiple domains, you could try adding this phrase [domain=""] to your query.
Example:
Query: as.org="Verizon"&&domain=""
